boot/zephyr/sample.yaml

```diff
@@ -0,0 +1,14 @@
+sample:
+  description: mcuboot test build sample
+  name: mcuboot
+common:
+    tags: introduction
+    harness: console
+    harness_config:
+      type: one_line
+      regex:
+        - "Hello World! (.*)"
```

Comment:
I think this part is from the hello_world sample and is not needed. The rest of the sample.yaml looks good to me. I will leave it up to you MCUboot developers if you want to have the header changes.

Reply:
of course - this is just a sandbox for now, will be fixed
Actually I would add at last test with configuration for serial recovery as well.